fp.js

```diff
@@ -32,3 +36,9 @@ var people = [
   {name: "Jasmine", age: 39}
 ]
 var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+
+var people = [
+  {name: "Jane", age: 32},
+  {name: "Jessica", age: 34},
+  {name: "Jasmine", age: 39}
+];people.forEach(peopleWhoseNamesHaveTheLetterS(person.name.match("s")));
```

Comment:
`forEach` does not return a collection. So in this question, you need to push each person that matches the condition into a new array.

```js
var people = { ... } // original data goes here
var peopleWhoseNamesHaveTheLetterS = [];
people.forEach(person => {
  peopleWhoseNamesHaveTheLetterS.push(person);
});
```